packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -202,6 +173,19 @@ export function TransferPanel() {
     connectionState
   })
 
+  function isWarningToken(): boolean {
+    const warningToken =
+      selectedToken && warningTokens[selectedToken.address.toLowerCase()]
+    if (warningToken) {
+      const description = getWarningTokenDescription(warningToken.type)
+      warningToast(
```

Comment:
With this we can just refactor this method to:
```typescript
function isWarningToken(token: ERC20BridgeToken | null) {
  if (!token) {
    return false
  }
  return typeof warningTokens[token.address.toLowerCase()] !== 'undefined'
}
```

Reply:
Also this could probably be a part of `TokenUtils`

Reply:
`warningTokens` is from `useAppState()` so unless further refactoring is done, it can't be moved out of a hook, so i wouldn't move it to the utils file